src/Framework/TestCase.php

```diff
@@ -1904,6 +1922,8 @@ protected function getMockForTrait(string $traitName, array $arguments = [], str
 
         $this->registerMockObject($mockObject);
 
+        Event\Registry::emitter()->testDoubleMockForTraitCreated($traitName);
```

Comment:
@sebastianbergmann

The DocBlock on `getMockForTrait()` currently uses `string` as type for `$traitName`.

Do you think it is worth changing this upstream to `trait-string`?

💁‍♂️ For reference, see https://psalm.dev/docs/annotating_code/type_syntax/scalar_types/#trait-string.

Reply:
Sure.

Reply:
Hang on a moment!